### DISCUSSION TITLE
Slightly different PK analysis in V10 and v11

### DISCUSSION THREAD

**Discussion by GhazalMontaseri** [Open-Systems-Pharmacology/Forum#1361] [Questions & Problems]
Dear OSP suite experts,
As I got it, there is no any model changes in V11 compared to V10. However, as an example, when I compared PK calculated parameters for 'iv 0.05 mg/Kg (2 min)' simulation from the PK-Sim file in V10 and V11 (Midazolam model in the OSP PBPK model library), there is a difference. The difference is not significant at all, but curious to know the origin for this difference in case there is no model change in V11.
![V10andV11](https://user-images.githubusercontent.com/30312003/197525744-836ad0f6-ab4d-453b-802f-dcc9074746c0.png)

**Comment by wilbertdew** [comment]
My guess is that this is due to random noise of the differential equation solver. Not sure about PK-Sim, but in MoBi the default relative tolerance is set to 1e-5, meaning only precision up to 5 significant digits.

**Comment by GhazalMontaseri** [comment]
Thanks, but do you thing precision up to 5 significant digits may lead to changes in PK parameters at the level of second digit after decimal?

**Reply by wilbertdew** [reply]
If there are 4 significant digits before the decimal, as in your example, yes. I just noticed that the Vd already has a difference in the 4th significant digit, before the decimal point, which I would not expect and for which I dont have an explanation.

**Comment by msevestre** [comment]
Can you share the V10 project?

**Comment by GhazalMontaseri** [comment]
the V10 project is here
https://github.com/Open-Systems-Pharmacology/OSP-PBPK-Model-Library/tree/v10.0/Midazolam

**Reply by msevestre** [reply]
ok perfect. I am going to check it out. We changed some settings in the way the solver works using derivatives so what I could imagine is that the output differs slightly but enough to change the PK ever so slightly. 
Numbers are saved as float with a precision of 6 to 9 digits so this should not be the problem here
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/floating-point-numeric-types

**Reply by StephanSchaller** [reply]
@msevestre , did you find out what the difference was cause by, here?
